dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/DSpaceConverter.java

```diff
@@ -15,7 +16,11 @@ public R convert(M source) {
 		return fromModel(source);
 	}
 
-	public abstract R fromModel(M obj);
+	public R fromModel(M obj) {
+		return fromModel(obj, DSpaceObjectRest.PRJ_DEFAULT);
```

Comment:
@abollini , the list converter you referenced makes sense to me.

I am curious to learn more about your thoughts on the 2 options that you have presented.

I imagine that each projection could use an alternative projection when loading related objects.  See https://github.com/DSpace-Labs/Rest7Contract/blob/master/projections.md.  Would this impact your thoughts for option 1?

For option 2, do we have the ability to detect lazy loaded fields from hibernate?  Wouldn't we trigger the loading at the point where we asked if the field was populated?  I would find some sample code helpful to understand this idea better.